### DISCUSSION TITLE
二次分发

### DISCUSSION THREAD

**Discussion by dabao1955** [RuriOSS/daijin#44] [Ideas]
您好，我想基于此项目进行二次分发以支持linux系统并添加上我的许可证，请问是否可以

项目地址(还没有创建):https://github.com/dabao1955/Moe-container-manager

**Comment by Moe-hacker** [comment]
> 您好，我想基于此项目进行二次分发以支持linux系统并添加上我的许可证，请问是否可以
> 
> 项目地址(还没有创建):https://github.com/dabao1955/Moe-container-manager

在遵守原许可内容的前提下，代码的修改与分发我无权干预，因此并不需要授权😉
